features/management_students/deleteStudent.feature

```diff
@@ -22,7 +22,7 @@ Feature: Set of tests to delete students
 			| userRole | adminUsername                | adminPassword  | studentUsername                  | studentPassword | deletedUserRole |
 			| admin    | kai.admin.qa@schul-cloud.org | Schulcloud1qa! | amelia.strobl.qa@schul-cloud.org | Schulcloud1qa!  | student         |
 
-	@deletedStudentIsNotVisibleInTeam @deletionConcept
+	@deletedStudentIsNotVisibleInTeam @deletionConcept @unstableTest
```

Comment:
if you add this test as unstable, you must remove the @stableTest on top, and make sure the other tests on this page have it

Reply:
done